prow/config/postsubmits.yaml

```diff
@@ -397,3 +397,27 @@ postsubmits:
           requests:
             cpu: "4"
             memory: 4Gi
+
```

Comment:
why postsubmit?

Reply:
to guarantee a healthy master branch?

Reply:
I know we're not really doing it for maistra, but most other projects using prow run tests again after merging

Reply:
I can remove it if you think we shouldn't run it-- this will likely be changed to a `push-xyz` job in the future anyway

Reply:
It doesn't guarantee a healthy branch. It won't revert the commit, will it?

What do you mean by 'push-xyz job in the future '?

Reply:
> What do you mean by 'push-xyz job in the future '?

What I'm building here is a WebAssembly filter. When there's a final draft for a packaging format upstream, I will likely add a postsubmit job that builds and pushes the package.


I'll remove the postsubmit for now

Reply:
Got it. Makes sense to me now. BTW, do you plan to move it under maistra org?